dist/index.html

```diff
@@ -0,0 +1 @@
+<!DOCTYPE html><html><head><meta charset="utf-8"><title>Battleship</title><script defer src="App.js"></script><link href="main.css" rel="stylesheet"></head><body><div id="app"></div></body></html>
```

Comment:
Read about `<main>` HTML tag. I think that you could change the `<div>` tag to `<main>` cause your app's main content gonna be shown in this element - always. It is more proper in terms of semantics IMO.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main